pkgs/development/python-modules/nipype/neurdflib.nix

```diff
@@ -0,0 +1,32 @@
+{ buildPythonPackage
+, fetchPypi
+, isodate
+, html5lib
+, SPARQLWrapper
+, networkx
+, nose
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "neurdflib";
+  version = "5.0.0";
+
+  src = fetchPypi {
+    inherit pname;
+    version = "5.0.0.post1"; # This is a packaging release. No code has been modified.
+    sha256 = "1qgmprixqxycxpjk9wjdmjykma14qqa2wcbx4nsldxi0ga7i7vv5";
+  };
+
+  propagatedBuildInputs = [ isodate html5lib SPARQLWrapper ];
+
+  checkInputs = [ networkx nose ];
+
+  # Python 2 syntax
+  # Failing doctest
+  doCheck = false;
+
+  checkPhase = ''
+    ${python.interpreter} run_tests.py
+  '';
+}
```

Comment:
`meta`?

Reply:
Why? It is not exposed in `python-packages.nix`.

Reply:
Someone might still be interested in e.g. the license, e.g. to block software with a certain license.